UPGRADE-5.3.md

```diff
@@ -23,6 +23,11 @@ HttpKernel
 
  * Marked the class `Symfony\Component\HttpKernel\EventListener\DebugHandlersListener` as internal
 
+Messenger
+---------
+
+* Deprecated `prefetch_count` parameter, it has no effect and will be removed in Symfony 6.0.
```

Comment:
Can you mention the concerned transport?

Reply:
:+1: